### ISSUE TITLE
Is there any accent neutralization plugin for the search feature?

### ISSUE THREAD

**Issue by morfin** [wenzhixin/bootstrap-table#1399]
Hi, 

Normally if you search for "José" by just typing "Jose", you won't get anything. Some neutralization has to be applied. 

I wonder if bootstrap-table supports this. 

datatables has this plugin:
(Replace accented characters with unaccented counterparts)
http://www.datatables.net/plug-ins/filtering/type-based/accent-neutralise

Any help will be appreciated.

Thanks.

**Comment by djhvscf** [comment]
Yes, we have this option data-strict-search, set to true and you will have this feature.

Example: http://jsfiddle.net/e3nk137y/3041/

**Comment by morfin** [comment]
Not exactly what I'm looking for.

Check this:
http://jsfiddle.net/e3nk137y/3045/

Try to search "Jose", without accent, and you won't get records with "José". 

That's what the accent-neutralization feature does.

Any other idea?

**Comment by djhvscf** [comment]
Ok, I understood in a wrong way.. Let me try to implement this.. @morfin

**Comment by morfin** [comment]
An "search-accent-neutralise" boolean option will be highly appreciated. Thanks.

**Comment by djhvscf** [comment]
See this: http://jsfiddle.net/e3nk137y/3055/ and this: https://github.com/wenzhixin/bootstrap-table/pull/1394

**Comment by morfin** [comment]
Wonderful.

**Comment by luisapuyen** [comment]
This solution was working for me until I tried it with a BootstrapTable that included the Filter Control plugin. For some reason, the search was getting mad and not neutralising accents.

Then I found out [here](https://github.com/wenzhixin/bootstrap-table/tree/master/src/extensions/accent-neutralise) in the documentation that I had to set  thie property

```
$table.bootstrapTable({
     searchAccentNeutralise: true
});
```

which is not set in the example in jsFiddle. Setting it to true got it working.

I'm not sure why it was working anyway without this property to true (default is false) in my previous BootstrapTables.
